bin/build-docker-image

```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/env bash
 
+if [ $# -eq 0 ] ; then
+  echo "Usage: $0 dockerhubusername"
```

Comment:
Yes, it does now (it used to only make the image locally). I'll add a more informative string here and detail it in the docs.